packages/@react-spectrum/combobox/stories/ComboBox.stories.tsx

```diff
@@ -64,7 +64,11 @@ let actions = {
   onInputChange: action('onInputChange'),
   onSelectionChange: action('onSelectionChange'),
   onBlur: action('onBlur'),
-  onFocus: action('onFocus')
+  onFocus: action('onFocus'),
```

Comment:
Being a good citizen and including all the possible events for Combobox and verified that they're being triggered in StoryBook.

Reply:
I like this, but it adds a lot of noise to all the stories. Could we target a few stories instead of all? Or add to all, but let stories turn them on?

Reply:
There are too many stories and I don't want to make assumptions about which events the stories need. I'll just add the one to the story I know would benefit from seeing the `onKeyDown` action event.